libsolidity/codegen/ir/IRGenerationContext.h

```diff
@@ -116,15 +116,7 @@ class IRGenerationContext
 	/// the code contains a call to an uninitialized function variable).
 	void internalFunctionCalledThroughDispatch(YulArity const& _arity);
 
-	/// Notifies the context that a direct function call (i.e. not through internal dispatch) was
-	/// encountered while visiting the AST. This lets the context know that the function should
-	/// not be added to the dispatch (unless there are also indirect calls to it elsewhere else).
-	void internalFunctionCalledDirectly(Expression const& _expression);
-
-	/// Notifies the context that a name representing an internal function has been found while
-	/// visiting the AST. If the name has not been reported as a direct call using
-	/// @a internalFunctionCalledDirectly(), it's assumed to represent function variable access
-	/// and the function gets added to internal dispatch.
+	/// Notifies the context that a function has been accessed but not called directly and adds the function to the internal dispatch.
 	void internalFunctionAccessed(Expression const& _expression, FunctionDefinition const& _function);
```

Comment:
Now that the other function is gone we could rename it to something like `addToInternalDispatch()`. It would make its purpose clearer.